backend/src/main/java/de/laura/project/api/service/TriviaApiService.java

```diff
@@ -0,0 +1,30 @@
+package de.laura.project.api.service;
+
+import de.laura.project.api.model.TriviaApiData;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Repository;
+import org.springframework.web.client.RestTemplate;
+
+@Repository
+
+public class TriviaApiService {
+
+    private final RestTemplate restTemplate;
+    private final String url = "https://opentdb.com/api.php?amount=10&category=17&difficulty=medium&type=multiple";
```

Comment:
you may want to consider including the base URL here to have e.g. categories passed through from frontend as path variables

Reply:
Yup, that was the plan, once I get it to work :). Thanks!